app/code/Magento/Customer/view/frontend/web/js/customer-data.js

```diff
@@ -86,7 +86,7 @@ define([
             var parameters;
 
             sectionNames = sectionConfig.filterClientSideSections(sectionNames);
-            parameters = _.isArray(sectionNames) ? {
+            parameters = _.isArray(sectionNames) && sectionNames.indexOf('*').length < 0 ? {
```

Comment:
Seems like it will not work:
![image](https://user-images.githubusercontent.com/1873745/87391205-f85e4680-c5b2-11ea-8665-b3de5bb9858b.png)
```suggestion
            parameters = _.isArray(sectionNames) && sectionNames.indexOf('*') < 0 ? {
```

Reply:
Typo fixed. Force of habit.